src/openforms/submissions/tests/test_tasks_pdf.py

```diff
@@ -262,6 +262,135 @@ def test_timestamp_included(self):
 
         self.assertEqual(reference_node.text, "Report created on: Jan. 1, 2024, 1 a.m.")
 
+    def test_list_fields_are_rendered_as_html(self):
```

Comment:
Good test - you can use this to change the implementation details :)

Reply:
I've changed it slightly. I've spread them out by content type (textfield, select, selectboxes and radio), to make it a bit more transparent what the expected behavior is